Apps/Common/src/Services/AuditService.cs

```diff
@@ -77,48 +81,45 @@ public AuditEvent ParseHttpContext(HttpContext context, AuditEvent audit)
         /// </summary>
         /// <param name="statusCode">The http context status code.</param>
         /// <returns>The mapped transaction result.</returns>
-        private AuditTransactionResult getTransactionResult(int statusCode)
+        private AuditTransactionResult GetTransactionResult(int statusCode)
```

Comment:
nit, private methods begin with lowercase

Reply:
I fixed a smell that was complaining this was lower case, I think the initial lower case only applies to fields not methods